graylog2-server/src/main/resources/org/graylog2/freemarker/templates/HTML/event_limit_reached.ftl

```diff
@@ -0,0 +1,9 @@
+<#if _title>
+    Event limit reached
+</#if>
+
+<#if _description>
+    <span>
+        Event limit »${event_limit}« reached for event definition »${event_definition_title}«. Try to use a more specific search query or use aggregations. Otherwise try to raise the limit.
```

Comment:
can we also show the eventdefinition id here? e.g. `title (6534de23f213ac)`

Reply:
fixed